src/Microsoft.CodeQuality.Analyzers/Core/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1343,4 +1343,10 @@
   <data name="AssigningSymbolAndItsMemberInSameStatementTitle" xml:space="preserve">
     <value>Assigning symbol and its member in the same statement.</value>
   </data>
+  <data name="AvoidInfiniteRecursionMessage" xml:space="preserve">
+    <value>This call is leading to an infinite recursion.</value>
```

Comment:
I think we need a better message here, something like `Do not set the property within the property setter`? @dotnet/roslyn-analysis for any suggestions.